src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -552,6 +552,27 @@ public function testRowSeparator()
         $this->assertEquals($expected, $this->getOutputContent($output));
     }
 
+    public function testAddRowsWithTableSeparator()
+    {
+        $table = new Table($output = $this->getOutputStream());
+        $table
+            ->addRow(new TableSeparator())
+            ->addRow(array('foo'))
+        ;
+        $table->render();
+
+        $expected =
+<<<TABLE
++-----+
++-----+
+| foo |
++-----+
+
+TABLE;
+
+        $this->assertEquals($expected, $this->getOutputContent($output));
```

Comment:
there are already tests for the rendering of table separators. This test should instead just  test the return value of `->addRow(new TableSeparator())`